Chrysalis-Papilio/Soen490-Papilio-Website
# Papilio 70 subtask - Email Address

## General info

The Android App user should be able to contact a business outside of the app, through their email address. The email was not asked to businesses prior to this PR so we had to implement it. This is a field added to the business creation form where business owner are entering information about their company. 

Issue number: Papilio-70

### Task description:

- Add business email input
- Add email to request body
- Modify test

## Manual Test
To test the email input you have to create your businessId then visit the profile form

Make sure that you have the appropriate variables in the backend .env file. (Contact me if you don't)

1. start the backend
```npm run build && npm start```
2. start the website
```npm start```
3. on the main page add the businessId then click on create business
4. Add information to the business email

##Testing

To run all the unit tests, run the following code: ```npm test```
To make sure that the tests have reach the appropriate level of coverage, run the following code: ```npm run coverage```

## Additional Notes
There are currently error in the merging of this branch with the main branch. This is correct. Those errors have been fixed in another branch PR #18. The parent branch of this branch is #18. 

**Do not merge before #18**
